src/common/routes/home/DsView.js

```diff
@@ -2773,11 +2783,15 @@ class DsView extends Component {
         }
     }
 
-    displayConnectedStatus () {
-        if (this.state.connectedState) {
-            return <span><i class='fas fa-server'></i> <b>Server connection:</b> <b style={{ 'color': 'darkgreen' }}>Connected</b></span>
+    displayConnectedStatus(){
+        if ( this.state.connectedState && this.state.dbconnectivitystate ) {
```

Comment:
We can check just the serverConnectedState. If that is down, then just server connectivity is down.
Once that check passes then only we can check for the dbConnectivityState. That way the code will make more sense.

Even though the current code will work perfectly but by reading code it's confusing a little bit.